charts/redpanda/templates/statefulset.yaml

```diff
@@ -250,8 +250,12 @@ spec:
                 - -c
                 - |
                   set -x
+                  {{- if ( dig "node" "recovery_mode_enabled" false .Values.config ) }}
```

Comment:
I would suggest to disable whole readiness probe, so that there will be no initial delay (via `initialDelaySeconds`).

Reply:
sure.